src/ol_concourse/pipelines/open_edx/xqwatcher/docker_packer_pulumi_pipeline.py

```diff
@@ -0,0 +1,170 @@
+import sys
+
+from bridge.settings.openedx.accessors import filter_deployments_by_application
+from bridge.settings.openedx.types import OpenEdxSupportedRelease
+
+from ol_concourse.lib.containers import container_build_task
+from ol_concourse.lib.jobs.infrastructure import packer_jobs, pulumi_jobs_chain
+from ol_concourse.lib.models.fragment import PipelineFragment
+from ol_concourse.lib.models.pipeline import (
+    GetStep,
+    Identifier,
+    Input,
+    Job,
+    Pipeline,
+    PutStep,
+)
+from ol_concourse.lib.resources import git_repo, registry_image
+from ol_concourse.pipelines.constants import PULUMI_CODE_PATH, PULUMI_WATCHED_PATHS
+
+
+def build_xqwatcher_pipeline(release_name: str):
+    xqwatcher_branch = OpenEdxSupportedRelease[release_name].branch
+    xqwatcher_repo = git_repo(
+        name=Identifier("openedx-xqwatcher-code"),
+        uri="https://github.com/openedx/xqwatcher",
+        branch=xqwatcher_branch,
+    )
+
+    xqwatcher_registry_image = registry_image(
+        name=Identifier("openedx-xqwatcher-container"),
+        image_repository="mitodl/xqwatcher",
+        image_tag=release_name,
+        username="((dockerhub.username))",
+        password="((dockerhub.password))",  # noqa: S106
+    )
+
+    xqwatcher_dockerfile_repo = git_repo(
+        name=Identifier("xqwatcher-dockerfile"),
+        uri="https://github.com/mitodl/ol-infrastructure",
+        branch="main",
+        paths=["dockerfiles/openedx-xqwatcher/Dockerfile"],
+    )
```

Comment:
Are we building a container for xqueue-watcher?

Reply:
🍝